packages/js/product-editor/src/components/woo-product-section-item/woo-product-section-item.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Slot, Fill } from '@wordpress/components';
+import { createElement, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+import { ProductFillLocationType } from '../woo-product-tab-item';
+
+type WooProductSectionItemProps = {
+	id: string;
+	tabs: ProductFillLocationType[];
+	pluginId: string;
+};
+
+type WooProductSectionSlotProps = {
+	tab: string;
+};
+
+export const WooProductSectionItem: React.FC< WooProductSectionItemProps > & {
+	Slot: React.FC< Slot.Props & WooProductSectionSlotProps >;
+} = ( { children, tabs } ) => {
+	return (
+		<>
+			{ tabs.map( ( { name: tabName, order: tabOrder } ) => (
+				<Fill
+					name={ `woocommerce_product_section_${ tabName }` }
+					key={ tabName }
+				>
+					{ ( fillProps: Fill.Props ) => {
+						return createOrderedChildren<
+							Fill.Props & { tabName: string }
+						>( children, tabOrder || 20, {
```

Comment:
I'm not sure it falls within the scope of this PR, if not, please disregard it, but how about moving 20 and other numbers to a constant?